mne/gui/_coreg.py

```diff
@@ -806,6 +828,16 @@ def callback(iteration, n_iterations):
             self._display_message(f"Fitting ICP - iteration {iteration + 1}")
             self._update_plot("sensors")
             self._current_icp_iterations = iteration
+            dists = self._coreg.compute_dig_mri_distances() * 1e3
+            self._status_msg.set_value(
+                "Distance between HSP and MRI (mean/min/max): "
+                f"{np.mean(dists):.2f} mm "
+                f"/ {np.min(dists):.2f} mm / {np.max(dists):.2f} mm"
+            )
+            self._status_msg.show()
+            self._status_msg.update()
+            self._renderer._status_bar_update()
+            self._update_distance_estimation()
```

Comment:
when you're done fitting it would be great to still have the last distances. They are displayed fast and at the end you have only "Fitting finished ...."

Reply:
What scenario do you suggest? Should I remove the benchmark completely? Should the last iteration never disappear? 
Let me know what you think is more useful

Reply:
By default I concatenated them in 96b67b2